### ISSUE TITLE
Deploy 1.4.2 to SDSC

### ISSUE THREAD

**Issue by keith923** [Electrostatics/apbs-pdb2pqr#185]
I'm going to get APBS installed on rocks-89 and then hand it off to @kmonson.

**Comment by sobolevnrm** [comment]
Any updates on this? @lizutah @kmonson

**Comment by keith923** [comment]
This got stalled because of #190.

**Comment by keith923** [comment]
Will we be able to close this once issue #372 is complete?  Do we need anything more than the APBS shared library for the python code?  Although, that's just for pdb2pka, right?  So I guess I need to know more about how these things run at SDSC.  Maybe @kmonson shore up my understanding?

**Comment by keith923** [comment]
BTW, if we do actually need an APBS roll, I'm going to ask Robert Konecny (rok@ucsd.edu) for help with it.

**Comment by keith923** [comment]
Just the Python lib won't do.  Kyle insists that Nadya will build the roll for us, and that pointing her at the source will suffice.

**Comment by keith923** [comment]
Let's test the release out on the APBS Mac to validate that it runs in a similar environment as SDSC.  @kozlac could you take care of this?

**Comment by lizutah** [comment]
Deployed in Feb.
